src/detect-engine.c

```diff
@@ -1250,14 +1203,6 @@ void DetectEngineBufferTypeSupportsFrames(DetectEngineCtx *de_ctx, const char *n
     SCLogDebug("%p %s -- %d supports frame inspection", exists, name, exists->id);
 }
 
-void DetectEngineBufferTypeSupportsPacket(DetectEngineCtx *de_ctx, const char *name)
```

Comment:
would like to keep this for completeness of the api

Reply:
ok